config/nav.yml

```diff
@@ -31,6 +34,9 @@ nav:
           - Sources, Brokers, and Triggers: getting-started/first-broker.md
           - Using a Knative Service as a source: getting-started/first-source.md
           - Using Triggers and sinks: getting-started/first-trigger.md
+        - More with Knative Functions:
```

Comment:
Why you have this split?
I think you should have the 4 pages under "Using Knative Functions" show the hello and echo back to back in one place.
On the navigation, if user clicks next it goes to Serving tutorial.

Reply:
Yep, splitting this way to me is very confusing.

Reply:
> Why you have this split?
> I think you should have the 4 pages under "Using Knative Functions" show the hello and echo back to back in one place.

This was meant as a compromise. I feel pretty strongly that functions create and deploy steps should come first, since they do not require any knative knowledge at all. Ashleigh did not like having some things like triggers introduced before the Knative Eventing section, so I split these out. Could we note in the section that discusses Triggers and such that the user might want to do that portion of the tutorial and then come back to Functions Next Steps? I don't think this way would be super clean either, but might be an improvement? WDYT?